frameworks/keyed/angular-cf-nozone/src/main.ts

```diff
@@ -1,10 +1,9 @@
 import { bootstrapApplication } from '@angular/platform-browser';
 import { AppComponent } from './app/app.component';
-import { NgZone, ɵNoopNgZone } from '@angular/core';
+import { provideExperimentalZonelessChangeDetection } from '@angular/core';
 
 bootstrapApplication(AppComponent, {
   providers: [
-    // https://github.com/angular/angular/issues/47538
-    { provide: NgZone, useClass: ɵNoopNgZone }
+    provideExperimentalZonelessChangeDetection()
```

Comment:
I would suggest reverting for now based on the discussion in https://github.com/pkozlowski-opensource/js-framework-benchmark/commit/307580b3c4bcd1cd83aef10bef3f075f3cd7f20f